### PR TITLE
[promptflow][internal] Integrate JS visualize bundle

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

For internal usage, integrate JavaScript visualize bundle to repository.

**Change List**:

- Update code to dump JavaScript bundle from package to where the HTML locates
- Remove code that uses CDN link
- Update Compliance Check YAML to ad-hoc delete this JavaScript bundle, otherwise the pipeline will fail.

The new HTML looks like:

image

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by RealKai42 — DISMISSED** [review]

File: src/promptflow/promptflow/_sdk/data/visualize.j2 [hunk 1]
```diff
@@ -10,6 +10,6 @@
     <script>
       window.bulk_test_details_data = {{ data }}
     </script>
-    <script src="{{ js_link }}"></script>
+    <script src="./bulkTestDetails.min.js"></script>
```
Comment (RealKai42) [inline src/promptflow/promptflow/_sdk/data/visualize.j2:13]: should use const `VIS_JS_BUNDLE_FILENAME` ?

Comment (zhengfeiwang) [reply src/promptflow/promptflow/_sdk/data/visualize.j2:13]: good catch, I forget to use that constant during template render

Comment (zhengfeiwang) [reply src/promptflow/promptflow/_sdk/data/visualize.j2]: updated to still use const during render, a minor update to rename to `js_path`

**Review by RealKai42 — APPROVED** [review]

File: src/promptflow/promptflow/_sdk/_constants.py [hunk 2]
```diff
@@ -110,10 +110,7 @@ def _prepare_home_dir() -> Path:
 DEFAULT_VARIANT = "variant_0"
 # run visualize constants
 VIS_HTML_TMPL = Path(__file__).parent / "data" / "visualize.j2"
-VIS_LIB_CDN_LINK_TMPL = (
-    "https://sdk-bulk-test-endpoint.azureedge.net/bulk-test-details/view/{version}/bulkTestDetails.min.js?version=1"
-)
-VIS_LIB_VERSION = "0.0.35"
+VIS_JS_BUNDLE_FILENAME = "bulkTestDetails.min.js"
```
Comment (crazygao) [inline src/promptflow/promptflow/_sdk/_constants.py:113]: why change url to file name?

**Review by crazygao — APPROVED** [review]

**Review by thy09 — APPROVED** [review]

**Pull request merged**